stores/searchHealthcareProfessionalStore.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from 'graphql-request'
+import { defineStore } from 'pinia'
+import { type Ref, ref } from 'vue'
+import { gqlClient } from '../utils/graphql.js'
+import type { HealthcareProfessional } from '~/typedefs/gqlTypes'
+
+export const useSearchHealthcareProfessionalStore = defineStore(
+    'SearchHealthcareProfessionalStore',
```

Comment:
let's make this camel case to match the other stores
```suggestion
    'searchHealthcareProfessionalStore',
```